sql/async_udfs.mdx

```diff
@@ -0,0 +1,115 @@
+---
+title: Asynchronous User-Defined Functions
+description: 'Run Async UDFs'
+---
+
+Arroyo's UDFs can be asynchronous.
+This is useful for UDFs that need to make network requests or perform other asynchronous operations.
+To define an asynchronous UDF, use the `async` keyword in the function definition.
+
+## Options
+
+Async UDFs have a few configuration options that can be defined in a special comment at the top of the UDF,
+similar to dependencies, under a `[udfs]` header.
+
+| Option | Example | Description |
```

Comment:
Might be cleaner to just have a `Default` column.

Reply:
The examples are the defaults so I just renamed the column